http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala

```diff
@@ -817,10 +817,13 @@ object RuntimeServiceInterp {
       config.autoFreezeConfig
     ) //TODO: use FiniteDuration for autopauseThreshold field in Cluster
     val clusterScopes = req.runtimeConfig match {
-      case Some(rq) if rq.cloudService == CloudService.GCE =>
-        if (req.scopes.isEmpty) config.gceConfig.defaultScopes else req.scopes
-      case Some(rq) if rq.cloudService == CloudService.Dataproc =>
-        if (req.scopes.isEmpty) config.dataprocConfig.defaultScopes else req.scopes
+      case Some(rq) =>
```

Comment:
This is probably scalafix, but maybe this is a little more concise:
```
    val clusterScopes =
      if (req.scopes.nonEmpty) req.scopes
      else
        req.runtimeConfig.map(_.cloudService) match {
          case Some(CloudService.Dataproc) => config.dataprocConfig.defaultScopes
          case _                           => config.gceConfig.defaultScopes
        }
```

Reply:
oops...forgot to fix this before merging...it wasn't scalafix, I didn't really read the code but was just mechanically fixing some syntex error

Reply:
np, not too important (i still approved the PR)